01-intro/README.md

```diff
@@ -57,7 +57,7 @@ _(Please note you are welcome to post under a pseudonym and/or password protect
 
 - Name -- [assignment title](assignment url)
 - David Leyva -- [No Paris???](https://wp.nyu.edu/davidaleyva/week-1-homework/)
-- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9)
+- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [](https://soojin-lee0819.github.io/ITPBlog/week1.html)
```

Comment:
Hi Soojin!

With markdown it's best if you can put a "title" for the link in between the square brackets: `[insert title here]`.

```suggestion
- Soojin -- [Converting English to Korean Using Numbers](https://www.notion.so/Programming-from-A-Z-7136b52629c945c0ae4c4c3e8dec03e6#43d3fadc094b4e92b54b732c49dbbff9) , [Translation with Numbers blog post](https://soojin-lee0819.github.io/ITPBlog/week1.html)
```